README.md

```diff
@@ -1,5 +1,9 @@
 # vizstorm-GIF
 
+# This needs scipiper task combiners
+
+This branch requires scipiper with task combiners!
```

Comment:
This should be a less important warning now that we've merged combiners into the main scipiper...but may as well leave it here.

Reply:
Although, likely not even true since we aren't actually using the `final_targets` in the make file.